definitions/features/foreman_openscap.rb

```diff
@@ -3,7 +3,7 @@ class Features::ForemanOpenscap < ForemanMaintain::Feature
     label :foreman_openscap
 
     confine do
-      check_min_version('tfm-rubygem-foreman_openscap', '0.5.3')
```

Comment:
Looks like 0.5.3 was released in January 2016. I'm guessing that's why it's no longer relevant to check for the version.

Reply:
Yes, that's what I thought